be/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,10 +15,10 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('fullname');
```

Comment:
- use `name` instead, it's up to the users if they want to put their fullname or just name. (please update ERD as well).